sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/UcfgInstructionTest.Assignment.cs

```diff
@@ -459,16 +459,29 @@ public void TestSimpleFieldAccessAndAssignment()
             const string code = @"
 namespace Namespace
 {
-    public class Class1
+    public class BaseClass
+    {
+        public string baseField;
+    }
+
+    public class Class1 : BaseClass
     {
-        public string Field;
+        public string field;
 
         public void Foo()
         {
-            var variable = Field;
-                // %0 := __id [ this.Field ]
+            var variable = field;
+                // %0 := __id [ this.field ]
                 // variable := __id [ %0 ]
-
+             variable = this.field;
+                // %1 := __id [ this.field ]
+                // variable := __id [ %1 ]
+             variable = baseField;
+                // %2 := __id [ this.baseField ]
+                // variable := __id [ %2 ]
+             variable = base.baseField;
```

Comment:
They are not directly fixed by this PR but I noticed we were not testing calls to `base`.